stdlib/crypto/src/main/java/org/ballerinalang/stdlib/crypto/nativeimpl/DecryptAesCbc.java

```diff
@@ -38,18 +36,7 @@
 
     @Override
     public void execute(Context context) {
-        BValue inputBValue = context.getRefArgument(0);
-        byte[] input = ((BValueArray) inputBValue).getBytes();
-        BValue keyBValue = context.getRefArgument(1);
-        byte[] key = ((BValueArray) keyBValue).getBytes();
-        BValue ivBValue = context.getRefArgument(2);
-        byte[] iv = null;
-        if (ivBValue != null) {
-            iv = ((BValueArray) ivBValue).getBytes();
-        }
-        String padding = context.getRefArgument(3).stringValue();
-        CryptoUtils.aesEncryptDecrypt(context, CryptoUtils.CipherMode.DECRYPT, Constants.CBC, padding, key, input, iv,
-                -1);
+
```

Comment:
Empty newline added to the end of most methods. I think this comes from formatting rule settings, but seems like the settings are incorrect.

Reply:
Yes. I fixed that with https://github.com/ballerina-platform/ballerina-lang/pull/15909/commits/b37b79a0fd639cba691569f560c9f3ca40b7c8fd and the `execute` method is to be removed once we fully migrated to JBallerina.